File: Notas de aula 5/Apresentação/while.js
oqueRepet = prompt("O quê deseja repetir?")
vezesRepet = prompt("Digite quantas vezes deseja repetir.")
if (Number.isFinite(vezesRepet)) {
    alert("Precisa ser um número.")
} else {
    while (vezesRepet == 2 * vezesRepet) {
        alert(oqueRepet);
        vezesRepet - 1
    }
}

File: Notas de aula 5/Apresentação/while.cpp
#include <iostream>
#include <typeinfo>
int main()
{
    using namespace std;
    string  oqueRepet;
    float vezesRepet, i;
    cout << "O quê deseja repetir?\n";
    cin >> oqueRepet;
    cout << "Quantas vezes deseja repetir?\n";
    cin >> vezesRepet;
    cout << typeid(vezesRepet).name << endl ;
    while (vezesRepet != 0) {
        cout << oqueRepet << endl;
        vezesRepet = vezesRepet - 1;
    }
    return 0;
}
